addon/components/polaris-text-field.js

```diff
@@ -504,21 +515,88 @@ export default Component.extend({
     return inputType === 'number' && !disabled;
   }).readOnly(),
 
-  checkFocus() {
-    let { input, focused } = this.getProperties('input', 'focused');
+  normalizedValue: computed('value', function() {
+    let value = this.get('value');
+    return value !== null ? value : '';
```

Comment:
Might be worth explicitly matching the React implementation and using `!=` here so that this catches `undefined` values etc.

Reply:
^this